Makefile.config.in

```diff
@@ -207,7 +207,6 @@ NATIVECCLIBS=@nativecclibs@
 SYSTHREAD_SUPPORT=@systhread_support@
 PACKLD=@PACKLD@
 FLEXDLL_CHAIN=@flexdll_chain@
-EXTRALIBS=@extralibs@
```

Comment:
IIRC `EXTRALIBS` was needed for an ancient version of the 64-bit Windows SDK, and will not be missed!